src/http/transport.rs

```diff
@@ -202,6 +202,7 @@ mod e2e_tests {
     }
 
     #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    #[ignore] // TODO: this is now failing due to changes made to `ByteArrStream`
```

Comment:
are there more changes expected for byte array stream or we can just delete this test?

Reply:
want to discuss with you. at the least, we'll want to change the test because this is the only e2e test right now for http

Reply:
is it possible to enable it as part of this PR or it is hard/not possible?

Reply:
as we discussed, I created a new `ByteArrStream`/`AlignedByteArrStream` dichotomy which better handles the abstraction between transport and query processor. see https://github.com/thurstonsand/raw-ipa/pull/12 for that change

Reply:
oh, and i did re-enable this test in #12